### PR TITLE
charts color changed, SummingTime component divided into smaller ones

### PR DESCRIPTION
**Pull request by lukaszgancarz98**

### CODE REVIEW

File: django_react_proj/users-fe/src/components/TaskProgressChart/TaskProgressChart.js [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    color: ['#34a017', '#ff0000'],
+    radius: 0.6,
+    label: {
+      type: 'spider',
+      content: '{percentage}',
+    },
+    interactions: [{ type: 'element-active' }],
+  };
+  return (
+    <>
```
Comment (bakowroc) [inline django_react_proj/users-fe/src/components/TaskProgressChart/TaskProgressChart.js:43]: Unnecessary fragment

File: django_react_proj/users-fe/src/components/TaskProgressChart/TaskProgressChart.js [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const TaskProgressChart = ({ users }) => {
+  let done = 0;
+  let notDone = 0;
+
+  if (users) {
+    users.forEach((user) => {
+      if (user.done === true) {
```
Comment (bakowroc) [inline django_react_proj/users-fe/src/components/TaskProgressChart/TaskProgressChart.js:12]: `if(user.done)`

File: django_react_proj/users-fe/src/App.js [hunk 3]
```diff
@@ -41,7 +46,31 @@ const App = () => {
         users={users}
       />
     ),
-    2: <Subtract users={getUserForView(users, currentDate)} workHours={workHours} />,
+    2: (
```
Comment (bakowroc) [inline django_react_proj/users-fe/src/App.js:49]: This whole block can be a separated component to make it better to maintain

**Review by bakowroc — APPROVED** [review]

File: django_react_proj/users-fe/src/components/TaskProgressChart/TaskProgressChart.js (hunk 1 above)
Comment (lukaszgancarz98) [reply django_react_proj/users-fe/src/components/TaskProgressChart/TaskProgressChart.js:43]: Deleted.

**Pull request merged**